mwrefs/utilities/fetch_references.py

```diff
@@ -28,12 +29,12 @@ def main(argv=None):
         level=logging.WARNING if not args['--debug'] else logging.DEBUG,
         format='%(asctime)s %(levelname)s:%(name)s -- %(message)s'
     )
-    run(args['<host>'], int(args['<rev-id>']))
+    run(args['<host>'], args['<title>'])
 
 
-def run(host, rev_id):
-    session = mwapi.Session("https://en.wikipedia.org")
-    doc = session.get(action='query', prop='revisions', titles='Anachronism',
```

Comment:
haha.  Looks like that was hard-coded as a title anyway.  We could use `revids=rev_id` here.